tests/test_gui.py

```diff
@@ -50,15 +62,25 @@ def test_gui():
     plotter.reset.button.on_clicked(plotter.reset_range(None))
 
 
-def test_gui_params():
-    plotter = read_chains('./tests/example_data/pc').gui()
-    assert len(plotter.param_choice.buttons.labels) == 8
+@pytest.mark.parametrize('root', ["./tests/example_data/pc",
+                                  "./tests/example_data/un"])
```

Comment:
Is there any reason this couldn't be extended to mn as well? It wouldn't be able to do mn_old, but this serves as encouragement for samplers to provide logL_birth contours so that you get access to this behaviour.

Reply:
Added in https://github.com/handley-lab/anesthetic/pull/313/commits/d6da757babc2c0e53f88edc9a01a7e954b0e9514 after all.